### ISSUE TITLE
Vanishing loot roll

### ISSUE THREAD

**Issue by MightyPhred** [WarEmu/WarBugs#17390]
My group just killed Lord Slaurith, he dropped a shaman and sorcerer belt, as you can see by the screenshot, 6 loot rolls are made on the sorcerer belt with the sorcerer being the predictable winner

for some reason, only 5 rolls are listed on the shaman belt, my roll of need disappeared and it was given to the greed roll
![loot](https://user-images.githubusercontent.com/16922842/88934332-6aec4900-d278-11ea-89be-1e0c98c3cb24.png)

**Comment by Kaelang** [comment]
Was you dead / left the group?

**Comment by MightyPhred** [comment]
Nope, took us several tries to kill him, when we pulled it off nobody died, also had I been dead it would have ignored both of my loot rolls, you can see I passed on one of the items
